Library/Homebrew/brew.sh

```diff
@@ -267,6 +271,12 @@ access to all bottles."
 EOS
 fi
 
+if [[ -z "$HOMEBREW_ENV_PRUNED" ]]
```

Comment:
I think that the double-guard was originally a ksh thing but it makes the test more obvious to me.